contrib/gocql/gocql/gocql.go

```diff
@@ -259,6 +269,16 @@ func (tb *Batch) newChildSpan(ctx context.Context) ddtrace.Span {
 }
 
 func (tb *Batch) finishSpan(span ddtrace.Span, err error) {
+	if err != nil && tb.params.config.errCheck != nil && tb.params.config.errCheck(err) == false {
+		// If the error is explicitely marked as not-an-error, then skip and ignore.
```

Comment:
This is a duplicate comment, which I see as a hint for the fact that it doesn't belong here, but more over at `WithErrorCheck`